cpp/evolution/v0/generated/yardl/detail/binary/coded_stream.h

```diff
@@ -1,358 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
Good catch.

When I removed them, I planned to change the "evolution" models to use the `internalSymlinkStaticHeaders` option like we did for the main test model.

Unfortunately, there's no way to symbolically link the `yardl/detail/` directory and have the files inside `#include "../../yardl.h", because "yardl.h" is *generated*.

This is why I completely removed the `internalSymlinkStaticHeaders` option for C++. FYI.

Reply:
No, there isn't.